### PR TITLE
Add new floodlight tracking alongside existing implementation

### PR DESCRIPTION
**Pull request by remybach**
🐿 v2.11.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "@financial-times/n-gage": "^2.0.2",
     "@financial-times/n-heroku-tools": "^7.4.0",
     "@financial-times/n-test": "^1.7.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
```
Comment (remybach) [inline package.json]: This was needed to get `n-ui` working with a locally running app.

File: components/n-ui/tracking/third-party/floodlight.js [hunk 2]
```diff
@@ -19,6 +19,9 @@ module.exports = function (flags) {
 	const isSignUpForm = /^\/signup/.test(location.pathname);
 	const isSubscriptionConfirmation = /^\/thank-you/.test(location.pathname);
 	const isTrialConfirmation = (l) ? l.getAttribute('data-signup-is-trial') === 'true' : undefined;
+	const isBarrier = !!document.querySelector('[data-trackable="page:product"]');
+	const isSubscriber = self.FT.flags.subscriberCohort;
```
Comment (liamkeaton) [inline components/n-ui/tracking/third-party/floodlight.js]: Where is self defined?

Comment (remybach) [reply components/n-ui/tracking/third-party/floodlight.js]: https://stackoverflow.com/questions/3638887/whats-the-difference-between-self-and-window 😉

Comment (liamkeaton) [reply components/n-ui/tracking/third-party/floodlight.js]: mindblown

I would say `window` is more well known and readable? `self` has been misused as scoped `this` for far too long.

File: components/n-ui/tracking/third-party/floodlight.js [hunk 3]
```diff
@@ -39,16 +42,33 @@ module.exports = function (flags) {
 		document.body.dispatchEvent(event);
 	};
 
-	if (flags && (flags.get('floodlight') && spoorId)) {
+	const gtagEvent = (sendTo) => {
+		self.gtag('event', 'conversion', {
```
Comment (liamkeaton) [inline components/n-ui/tracking/third-party/floodlight.js]:
```suggestion
		window.gtag('event', 'conversion', {
```

**Pull request merged**